packages/fxa-auth-server/scripts/test-local.sh

```diff
@@ -10,7 +10,7 @@ if [ -z "$CORS_ORIGIN" ]; then export CORS_ORIGIN="http://foo,http://bar"; fi;
 
 set -u
 
-DEFAULT_ARGS="--require ts-node/register --recursive --timeout 5000 --exit"
+DEFAULT_ARGS="--require ts-node/register --recursive --timeout 50000 --exit"
```

Comment:
I think we'll want to return this back to its original timeout amount -- going from 5 to 50 seconds is a big change. What was it for?

Reply:
Ahhh yes I forgot to revert this change. All the tests were timing out for me so I had to change it to make em work